docs/src/modules/ROOT/pages/constraints-and-score/performance.adoc

```diff
@@ -4,35 +4,44 @@
 [#scoreCalculationPerformanceTricksOverview]
 == Overview
 
-The `Solver` will normally spend most of its execution time running score calculation,
+The `Solver` will normally spend most of its execution time evaluating moves and running score calculation,
 which is called in its deepest loops.
-Faster score calculation will return the same solution in less time with the same algorithm,
+Faster move evaluation will return the same solution in less time with the same algorithm,
 which normally means a better solution in equal time.
 
 
-[#scoreCalculationSpeed]
-== Score calculation speed
+[#moveEvaluationSpeed]
```

Comment:
Have you checked all the references to this?
Antora has a habit of failing when refs don't exist.

Also, I'd call this section "Move evaluation and score calculation speed".

Reply:
Yes, I did check the references.